src/modules/35-connectors/components/CreateConnector/commonSteps/ConnectivityModeStep/ConnectivityModeStep.tsx

```diff
@@ -149,7 +149,7 @@ const ConnectivityModeStep: React.FC<StepProps<ConnectorConfigDTO> & Connectivit
           formName={`connectivityModeForm${props.type}`}
           enableReinitialize
         >
-          {formik => {
+          {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
          {(formik: unknown) => {
```